play-dl/YouTube/classes/LiveStream.ts

```diff
@@ -163,13 +163,12 @@ export class Stream {
     }
 
     private cleanup(){
-        clearInterval(this.timer)
+        clearTimeout(this.timer as NodeJS.Timer)
         this.request?.unpipe(this.stream)
         this.request?.destroy()
         this.request = null
+        this.timer = null
         this.url = ''
-        this.bytes_count = 0
-        this.per_sec_bytes = 0
```

Comment:
clearInterval to clearTimeout and remove about `this.bytes_count` and `this.bytes_count`

Reply:
This will have no effect since I am adding that bytes_count condition which will trigger at first run of loop function.

Reply:
Oh now I see what are you trying to do.